other/materials_designer/Introduction.ipynb

```diff
@@ -29,8 +29,9 @@
     "### 1.4. 0D Structures.\n",
```

Comment:
<p>1.4.2. Create a cluster by cutting it out of a slab.</p><p>1.4.3. Create a cluster by layering a specific shape.</p><p> </p><p>We could highlight that in one case it's "subtractive" - cut out of a slab, in the other - "additive" by building in layers of a specific shape. But </p><p> </p><p>Don't forget dots at the end.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>